### PR TITLE
Fixed discord last update

### PR DESCRIPTION
**Pull request by G4brym**
Last update requires the string Bot before the token

### CODE REVIEW

File: cmd/bot/bot.go [hunk 1]
```diff
@@ -709,7 +711,7 @@ func main() {
 
 	// Create a discord session
 	log.Info("Starting discord session...")
-	discord, err = discordgo.New(*Token)
+	discord, err = discordgo.New(TOKEN)
```
Comment (b1naryth1ef) [inline cmd/bot/bot.go]: No need to create another variable, just:

```go
discord, err = discordgo.New("Bot " + *Token)
```

**Comment by b1naryth1ef** [comment]
I just adjusted the token passed in on the command line, but this is totally reasonable to merge.

**Comment by G4brym** [comment]
I tried that, but got an error, golang is not my best skill

**Comment by G4brym** [comment]
but yes, it now works that way

**Comment by reyqn** [comment]
Why isn't this merged?

**Pull request closed without merging**